src/utils.jl

```diff
@@ -186,13 +186,9 @@ end
 Return rotation matrix from vector `u` to vector `v`.
 """
 function uvrotation(u::Vec{2}, v::Vec{2})
-  convert(DCM, CounterClockwiseAngle(∠(u, v)))
+  Angle2d(∠(u,v))
 end
 
 function uvrotation(u::Vec{3}, v::Vec{3})
-  u⃗ = normalize(u)
-  v⃗ = normalize(v)
-  re = √((1 + u⃗ ⋅ v⃗) / 2)
-  im = (u⃗ × v⃗) / 2re
-  convert(DCM, Quaternion(re, im...))
+  return rotation_between(v, u)
```

Comment:
Nice, it would be great if we could remove uvrotation in favor of rotation_between, also is there a reason for the order (v, u) instead of (u, v)?